### DISCUSSION TITLE
Calculated fields in table catchment_area

### DISCUSSION THREAD

**Discussion by urskaufmann** [QGEP/QGEP#824] [Ideas]
While working with catchment_area, I miss the values "reduzierte Fläche" ( 4 times!)  or number of inhabitants for the selected catchment_areas.
I think there are two ways to get to the calculate values:
- create a view with this calculated values and work in qgep with the view instead of the table
- stay with the table and configure this calculated fields as virtual fields in the qgep-project

Which way is the TEKSI - way?

**Comment by ponceta** [comment]
Thank you for this good question. As discussed with @sjib we think the answer is in the question :

"TEKSI way" would be :
- Provide a prototype based on calculated virtual fields with formulas and screenshots
- Challenge the community on the usage -> mutualize the needs, check for genericity
-  Implement (self or submitted to a developper) the approach in the best format  for the use case (psql view, constants stored in a sys table, ... ) 

Views will always be easier to maintain on the long term on such a solution. (generic and community based)
